axlearn/cloud/gcp/tpu_health_check.py

```diff
@@ -31,6 +31,7 @@
 from contextlib import contextmanager
 from datetime import datetime
 from typing import Literal, Optional, Union
+import pytest
```

Comment:
is this import necessary?

Reply:
https://github.com/ptoulme-aws/axlearn/blob/52d0991ed0224f034ccf82b088d7e39b112cd302/axlearn/cloud/gcp/tpu_health_check.py#L65

Reply:
we could potentially import thorugh abseil library, would have to check